src/lib/replication/repl_dev/raft_state_machine.cpp

```diff
@@ -141,14 +165,14 @@ repl_req_ptr_t RaftStateMachine::transform_journal_entry(nuraft::ptr< nuraft::lo
         auto new_buf = nuraft::buffer::expand(*rreq->raft_journal_buf(),
```

Comment:
No, but in other use cases (not HomeObject), blocks allocated are not contiguous (so MultiBlkId could be more than one entry). It could be different on different replicas right? So one proposer it could be contiguous blocks and hence serialized of the BlkId is just 1 byte, whereas it could be multiple in one follower which could 4 blocks and hence 6 bytes. Here we replace them by allocating a fresh buffer and copy them (it should be rare but can happen). For HomeObject use case, this code block will not be excercised.